.github/dependabot.yml

```diff
@@ -1,10 +1,23 @@
-
 version: 2
 updates:
 - package-ecosystem: cargo
   directory: "/"
   schedule:
-    interval: daily
+    interval: "weekly"
+    timezone: "Etc/UTC"
+    day: "Saturday" # so we can approve and merge over the weekend, while less is going on
+    time: "5:30"
```

Comment:
This might work but the docs say the format is `hh:mm`

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduletime